pkg/ctl/create/cluster.go

```diff
@@ -216,6 +216,9 @@ func doCreateCluster(cmd *cmdutils.Cmd, ngFilter *filter.NodeGroupFilter, params
 		if err := importSubnets(cfg.VPC.Subnets.Private, params.Subnets[api.SubnetTopologyPrivate]); err != nil {
 			return err
 		}
+		if params.DryRun {
+			cfg.AvailabilityZones = nil
+		}
```

Comment:
wow this was a small change :D nice one
Can we add a test for this or update docs?

Reply:
We need to add an integration test (https://github.com/weaveworks/eksctl/issues/5412), but not as part of this PR.